### PR TITLE
Added azure release 13.0.0-beta1

### PR DESCRIPTION
**Pull request by whites11**
This PR is still a WIP

### CODE REVIEW

File: azure/v13.0.0-beta1/release.yaml [hunk 1]
```diff
@@ -19,40 +19,41 @@ spec:
...
-  - name: coredns
-    version: 1.6.5
+    version: 2605.7.0
   - name: calico
     version: 3.15.3
   - name: etcd
-    version: 3.4.9
-  date: "2020-10-27T11:04:44Z"
+    version: 3.4.13
+  - name: chart-operator
```
Comment (fiunchinho) [inline azure/v13.0.0-beta1/release.yaml:53]: `chart-operator` is already listed as `app`. I guess it only needs to be added once?

Comment (whites11) [reply azure/v13.0.0-beta1/release.yaml:53]: maybe, let's compare with latest AWS release

File: azure/v13.0.0-beta1/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+This is the first release to support Kubernetes 1.18 on Azure.
+
+This release also includes support for Kubernetes node pools.
+A node pool is a subset of the Kubernetes nodes. They enable having pools of nodes with different configurations (like a different instance size) within one cluster.
+After cluster creation with 1 node pools, additional node pools can be freely added and removed from the cluster.
+
+If you have access to the Control Plane API you can manage your clusters directly from there.
+The clusters that you create are now represented by [Cluster API](https://cluster-api.sigs.k8s.io/) CRDs ([Custom Resource Definition](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)).
+Using [our kubectl plugin](https://github.com/giantswarm/kubectl-gs/) you can easily create the Custom Resources required to create a cluster.
+
```
Comment (MartaD) [inline azure/v13.0.0-beta1/README.md:12]:
```suggestion

This tenant cluster release is marked as beta, which means that it contains all the content of the future v13.0.0 stable release and secures upgrade paths from and to that release. Beta version is recommended to be used and tested in non-production clusters. The only difference between this beta version and the stable one will be bug fixes for issues found during your testing.

```
wdyt?

Comment (whites11) [reply azure/v13.0.0-beta1/README.md:12]: Sounds good, merging

Comment (tuommaki) [reply azure/v13.0.0-beta1/README.md:12]: Works for me.

File: azure/v13.0.0-beta1/release.yaml (hunk 1 above)
Comment (whites11) [reply azure/v13.0.0-beta1/release.yaml]: removed

**Review by tuommaki — APPROVED** [review]

**Review by MartaD — APPROVED** [review]

**Pull request merged**